pages/en/blog/announcements/diving-into-the-nodejs-website-redesign.md

```diff
@@ -0,0 +1,118 @@
+- [README](https://github.com/nodejs/nodejs.org/blob/main/README.md)
+- [TRANSLATION](https://github.com/nodejs/nodejs.org/blob/main/TRANSLATION.md)
+
+New code has a strong focus on inline code and configuration comments, separation of concerns, and clearly defined constants. The use of TypeScript throughout helps contributors understand the shape of data and the expected behavior of functions.
+
+## Future Plans
+
+The redesign sets the stage for a new era of the Node.js website. We have a solid foundation in place that will last for years to come. If the past is any indicator, we'll be iterating within this space for a long time.
+
+But the work isn't done yet. We'll:
+
+- extend the site redesign to the API docs. They are in a separate codebase but plan to port the styles developed here to the API. This is careful work again that cannot disrupt daily use or contribution.
+- explore a monorepo for the website and API docs. This should improve coupling where it matters and reduce the overhead of managing two separate codebases.
+- reset internationalization efforts. The prior translations could not be carried over. Our heavy markdown / MDX approach poses a unique challenge we are partnering with Crowdin to solve.
+- continue to improve the CI/CD processes. We've made great strides in providing self-service feedback to contributors, but there's more to do.
+
+## Thanks
+
+Many people and organizations have contributed in both big and small ways to realize the redesign. We'd like to thank:
+
+- first and foremost, all [contributors and collaborators](https://github.com/nodejs/nodejs.org/graphs/contributors) that make this project possible.
+- [Chromatic](https://www.chromatic.com/) for providing the visual testing platform that helps us review UI changes and catch visual regressions.
+- [Cloudflare](https://cloudflare.com) for providing the infrastructure that serves Node.js's Website, Node.js's CDN, and more.
+- [Crowdin](https://crowdin.com/) for providing a platform that allows us to localize the Node.js Website and collaborate with translators.
+- [Hayden Bleasel](https://haydenbleasel.com/) for his design work on the Node.js redesign.
+- [Orama](https://docs.oramasearch.com/) for providing a search platform that indexes our content and provides lightning-fast results.
+- [Sentry](https://sentry.io/welcome/) for providing an open source license for their error reporting, monitoring, and diagnostic tools.
+- [Vercel](https://www.vercel.com/) for providing the infrastructure that serves and powers the Node.js Website
+- and lastly, the [OpenJS Foundation](https://openjsf.org/) for their support and guidance.
+
+In closing, Matteo Collina, a member of the TSC, said this on social media:
+
+> Here is your weekly reminder that the Node.js project is driven by volunteers. Therefore every feature that lands is because someone spent time (or money) to make it happen. This is called Open Governance.
```

Comment:
```suggestion
> Here is your weekly reminder that the Node.js project is driven by volunteers. Therefore every feature that lands is because someone spent time (or money) to make it happen. This is called Open Governance.
> <cite>"Matteo Collina"</cite>
```

We should try to use our styled `cite`

Reply:
landed via https://github.com/nodejs/nodejs.org/pull/6524/commits/03f70eb5ccffe3c4dd68a7945a61b407c01d4b66

Reply:
May I get a backlink to the original tweet or my twitter profile?

https://x.com/matteocollina/status/1770496526424351205?s=46&t=22eoAstJVk5l46KQXYEk5Q

Note that you can also embed it directly, however you prefer.